src/role-access/services/role-assignment.service.ts

```diff
@@ -0,0 +1,20 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { first } from 'rxjs/operators';
+import { Role } from '../models/role.model';
+
+@Injectable({ providedIn: 'root' })
+export class RoleAssignmentService {
+  public static allocationsUrl = '/api/role-assignment';
+
+  public judicialRoles: Role[];
+  public legalOpsRoles: Role[];
+  constructor(private readonly http: HttpClient) { }
+
+  public setRoleAllocations(): void {
```

Comment:
Could use `forkJoin` and avoid multiple subscribes. However this doesn't look right why is the service method subscribing to itself.

Reply:
Done